src/test/resources/alma-fix/99370694196806441.json

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
is it necessary to add this resource or does it structurally just reflects what other test resources already do? If you just added this resource because you mentioned it in the issue I suggest to change the example in the issue and remove this test file here to keep the files as small as reasonable.

Reply:
I appreciate the rationale and, no, it's technically not necessary in the context of this pull request (there are other test files illustrating the effect). But it's the only test record, AFAICS, that shows the effect of the new NWBib filter (3a84d88) in combination with this change.

Reply:
good then :+1: